mailserver/mailserver.go

```diff
@@ -260,7 +273,7 @@ func (s *WMailServer) DeliverMail(peer *whisper.Peer, request *whisper.Envelope)
 	)
 
 	if batch {
-		requestsBatchedCounter.Inc(1)
+		requestsCounter.WithLabelValues("batch").Inc()
```

Comment:
I would make a separate counter for this one.

Reply:
Done.